src/redux/books/books.js

```diff
@@ -0,0 +1,29 @@
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+
+const initialState = [];
+
+export const addBook = (payload) => ({
+  type: ADD_BOOK,
+  payload,
+});
+
```

Comment:
- [ ] You have implemented the Book actions and reducers. Well done. However, when I attempt to add a new book, the page reloads and no book is added. The reason is that you simply created the `addBook()` action but you are not dispatching it yet.